spec/features/donuts/01_user_can_view_a_list_of_donuts_spec.rb

```diff
@@ -21,13 +22,14 @@
   end
 
   scenario "clicks link and is taken to show page for a given donut" do
```

Comment:
What link? When you wrote this test originally, the only link was for the show page. Now you added a nav bar, and suddenly 'clicks link' is not explicit. This is a prime example of why making tests overly explicit at the start is a good idea for your code to remain explicit as it builds on complexity :)